CONTRIBUTING.md

```diff
@@ -67,7 +67,7 @@ following steps when creating a PR:
 2. The branch can contain any number of commits.  When merged, all commits will
    be squashed into a single commit.
 3. The changes should be thoroughly tested.  Please refer to [this
-   document](https://github.com/nearprotocol/nearcore/wiki/Writing-tests-for-nearcore)
+   document](https://github.com/near/nearcore/blob/master/docs/practices/testing/README.md)
```

Comment:
Please consider referring to a specific commit instead of `master`.  That way if the page gets moved, the link will not break.

Reply:
Good point.  OK, then this setup should be preferred.